scss/techs.scss

```diff
@@ -5,39 +5,20 @@ $card-height: 350px;
 $card-width: 210px;
 
 .techs-container {
-  display: flex;
-  flex-grow: grow;
-  margin-left: 160px;
-}
-
-.main-tech-container {
-  margin-right: 30px;
-  position: relative;
-
-  &:hover {
-    .background-tech-container {
-      left: 10px;
-      top: 10px;
-    }
-  }
+  align-items: center;
+  display: grid;
+  grid-column-gap: 20px;
+  grid-template: repeat(2, $card-height) / repeat(3, $card-width);
+  justify-content: center;
 }
 
-.background-tech-container {
-    border: 1px solid;
-    border-color: $green;
-    border-radius: 5px;
-    height: $card-height;
-    left: -10px;
-    position: absolute;
-    top: -10px;
-    transition: left 0.5s, top 0.5s;
-    width: $card-width;
-    z-index: -1;
+.first-tech-container {
+  border-right: 2px $green solid;
+  grid-row: 1 / span 2;
```

Comment:
This should be `1 / 3`

Reply:
Also, keep in mind the first tech should have more height than the one it has now

Reply:
Done